test/gui/.pylintrc

```diff
@@ -39,10 +37,13 @@ unsafe-load-any-extension=no
 
 [BASIC]
 attr-naming-style=snake_case
+; TODO: enable it later
+; class-attribute-naming-style=snake_case
```

Comment:
looks like it requires a bit of restructuring class variables. So, I will enable it later (next PR)